pkgs/games/flightgear/default.nix

```diff
@@ -94,7 +95,7 @@ stdenv.mkDerivation rec {
     udev
     fltk13
     apr
-    qtbase
+    qt5.full
```

Comment:
There's no package in Nixpkgs that uses the full qt5 set like that. Did you try adding only the needed modules first?

Reply:
done, not sure how launcher was working before without required libs